data/json/items/armor/bespoke_armor/custom_boots.json

```diff
@@ -63,34 +62,22 @@
   {
     "id": "boots_xlsurvivor",
     "type": "ARMOR",
-    "category": "armor",
```

Comment:
I'm not seeing a copy from for this item.  Am I missing it?

Reply:
I'll add it wasn't there before either.

Reply:
One hunk down outdate this.

Reply:
Added to test.